virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc

```diff
@@ -1,21 +1,44 @@
-[id="virt-managing-offline-vm-snapshots"]
-= Managing offline virtual machine snapshots
+[id="virt-managing-vm-snapshots"]
+= Managing virtual machine snapshots
 include::modules/virt-document-attributes.adoc[]
-:context: virt-managing-offline-vm-snapshots
+:context: virt-managing-vm-snapshots
 
 toc::[]
 
-You can create, restore, and delete virtual machine (VM) snapshots for VMs that are powered off (offline). {VirtProductName} supports
-offline VM snapshots on:
+You can create, restore, and delete virtual machine (VM) snapshots for VMs, whether the VMs are powered off (offline) or on (online). {VirtProductName} supports offline VM snapshots on:
 
 * Red Hat OpenShift Container Storage
 * Any other storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
 
+[NOTE]
+====
+Online snapshots are not supported for hot-plugged virtual disks.
```

Comment:
Online snapshots are not supported for virtual machines that have hot-plugged virtual disks.

Reply:
Got it. Thanks.